src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -79,14 +82,14 @@ public StartsEC2(final Amazon amaz) {
     @Override
     //@todo #629 Add Instance params to Directive, for example publicIpAddress
     public Iterable<Directive> process(final XML xml) throws IOException {
-        final Instance instance = this.amazon.runOnDemand();
+        final ContainerInstance instance = this.amazon.runOnDemand();
```

Comment:
@xupyprmv the problem here is that `ContainerInstance` needs to be this http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ecs/model/Container.html instead :)